python/folderify/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           python 1.0
+
+github.setup        lgarron folderify 1.2.3 v
+github.tarball_from archive
+
+categories          python amusements
```

Comment:
The python category is for modules one might include for use in other Python software, not for end-user software that just happens to be written in Python. (We don't have a c category for software that happens to be written in C, for example.)